apps/prairielearn/src/pages/authLogout/authLogout.ts

```diff
@@ -20,7 +19,7 @@ router.get(
       res.cookie('pl_disable_auto_authn', '1');
     }
 
-    await util.promisify(req.session.destroy);
+    await req.session.destroy();
```

Comment:
This was actually previously broken, as we never called the function returned from `util.promisify`.